config.yaml

```diff
@@ -6,8 +6,6 @@ service:
     port:
 packages:
     rpms:
-        - bash
-        - coreutils
```

Comment:
Why are we dropping these packages from config.yaml? It's not explained in the commit message. "to support new version of MTF" is jargon and doesn't explain why we are removing two lines.